dev-php/pecl-yaml/files/pecl-yaml-2.2.3-c99-incompatible-pointer-types.patch

```diff
@@ -0,0 +1,22 @@
+From e7bffc01c496ef36ce672c612984b13a27426788 Mon Sep 17 00:00:00 2001
```

Comment:
Include a link to the patch source (the upstream commit) + the Gentoo bug at the top.